minmea.c

```diff
@@ -612,25 +612,33 @@ bool minmea_parse_zda(struct minmea_sentence_zda *frame, const char *sentence)
   return true;
 }
 
-int minmea_gettime(struct timespec *ts, const struct minmea_date *date, const struct minmea_time *time_)
+int minmea_getdate(struct tm *tm, const struct minmea_date *date, const struct minmea_time *time_)
```

Comment:
thoughts on how to avoid the variable naming that matches the struct naming? In particular 'tm' vs. 'tm'

Reply:
Fixed to follow existing convention.